api/management/commands/import_admin2s_csv.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+import csv
+from django.core.management.base import BaseCommand, CommandError
+from api.models import District, Admin2
+
+
+class Command(BaseCommand):
+    help = "Import admin2 data from CSV"
+    missing_args_message = "Filename is missing. Filename / path to CSV file required. Required headers in CSV: distr_code, GOadm2code, ADM2"
+
+    def add_arguments(self, parser):
+        parser.add_argument('filename', nargs='+', type=str)
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        filename = options['filename'][0]
+        with open(filename) as csvfile:
+            reader = csv.DictReader(csvfile)
+            for row in reader:
+                print(row)
+                district_code = row['distr_code']
+                district = District.objects.get(code=district_code)
+                print(district.name)
+                adm2_code = row['GOadm2code']
+                try:
+                    adm2 = Admin2.objects.get(code=adm2_code)
+                except:
+                    print('HELLO')
```

Comment:
@szabozoltan69 @batpad we should have removed this before the merge

Reply:
Too late, deployment is in progress.

Reply:
Thanks for the catch @frozenhelium - removed in https://github.com/IFRCGo/go-api/pull/1376